ptr.py

```diff
@@ -192,8 +192,13 @@ def _analyze_coverage(
 
     failed_output = "The following files did not meet coverage requirements:\n"
     failed_coverage = False
+
     for afile, cov_req in required_cov.items():
-        if coverage_lines[afile].cover < cov_req:
+        try:
+            cover = coverage_lines[afile].cover
+        except KeyError:
+            raise KeyError("{} doesn't exist while in required coverage.")
```

Comment:
So close. Let's just error a friendly line here and not re-raise the exception. Getting rid of the ugly exception is what we want here.
```python
        except KeyError as ke:
            LOG.error("{} has not reported any coverage. Does the file exist? Does it get ran during tests? ({})".format(afile, str(ke)))
            return None
```

- Please remember to split the string and black format the file 💃